en/guide/crc.md

```diff
@@ -0,0 +1,22 @@
+# MAVLink CRC
+
+MAVLink code that needs to use a Cyclic Redundancy Check (CRC) should choose the CRC32 implementation described below.
+
+> **Note** Using the same CRC implementation for all cases means that only one implementation is required. 
+  Do not introduce another unless there without a compelling technical reason.
+
```

Comment:
Add another note here saying:

```
> **Note** The CRC used internally for the MAVLink serialization is not related to the CRC32 described here which is solely for user facing APIs on top of MAVLink.
```

Reply:
Thanks very much for review. I have fixed up somewhat like this :-)